config/routes/api.rb

```diff
@@ -8,7 +8,7 @@
     end
 
     resources :users, only: [:show, :create, :update, :destroy], constraints: { id: /.*/ }
-    resources :notes, only: [:index, :create, :update] do
+    resources :notes, only: [:index, :create, :update, :destroy] do
```

Comment:
Use `%i` always. Make it a practice. Otherwise it wouldn't come up naturally while coding. [Refer](https://www.bigbinary.com/learn-rubyonrails-book/adding-a-new-task-using-create-action#bb-generated-p-id-33).

Also, here we can use `except` over `only` given that except will have fewer items.